shell/client/shell.scss

```diff
@@ -616,6 +616,12 @@ body>.popup {
     >*:last-child {
       margin-bottom: 16px;
     }
+
+    &.centered {
+      margin: 0px auto;
+      width: 600px;
+      margin-left: calc(50% - 300px);
```

Comment:
You set margin left/right to `auto` above, and now you're setting it to calc. Since the element is `position: fixed` (on desktop), I think you actually don't want to do anything with margin here, but instead want: `left: calc(50% - 300px);`

It also seems like this styling is not what you want on mobile, so perhaps it should be moved into the desktop-specific section above?